roslibrust_codegen/src/lib.rs

```diff
@@ -0,0 +1,230 @@
+use proc_macro2::TokenStream;
+use quote::{format_ident, quote};
+use roslibrust_rospack as rospack;
```

Comment:
I'm a little leery of using the rospack name given that rospack does a bunch of things we don't do, and I don't want to imply that were publishing a crate that will eventually support all rospack functionality.

Maybe this is okay, but gut reaction was roslibrust_util could be a better name...

Also I'm starting to look ahead to what ros2 support will look like and not sure if we would need a ros1pack vs ros2pack...

Reply:
I don't think we necessarily need to support everything that `rospack` currently does. But `rospack` does a lot of things that will be useful and using via shelling out or binding the C++ library will be a huge nuisance. If I was using `roslibrust` and needed to do something that `rospack` does, naming it explicitly that seems like it'd help me find it.

On the other hand, it is a little awkward if we define some functions that aren't done by `rospack`. Not sure where I fall on it.

Reply:
Good enough for now!